src/03-arithmetic-expressions/arithmetic-expressions.md

```diff
@@ -19,7 +19,7 @@ call Identifier, or Id.
 \\]
 
 
-```scala mdoc
+```scala
```

Comment:
With scala mdoc the Scala code is automatically checked for errors, so we should use that for all code blocks. Or is there some reason, why you didn't use it?

Reply:
For now I dont see the possibility to uncomment lecture notes in between the scala blocks without raising an error, for example almost all of the 3rd chapter is embedded by curly braces in Scala code:
```scala
object AEId {
...
}
```
so mdoc raises an error for each specific block, before its put together (or so it seems).

Reply:
Hmm, I thought mdoc would produce one Scala file from the Markdown file and then check as a whole... I have to play around with this a bit.